draft-ietf-quic-multipath.md

```diff
@@ -616,7 +592,36 @@ be controlled by the combination of one or several of the following:
    uses a series of consecutive sequence numbers without creating
    holes.
 
-### ACK Delay Considerations
+### NULL-CID-Loss-and-Congestion
+
+Implementations MUST implement per-path congestion control. Congestion
```

Comment:
I don't think per-path congestion control is specific to NULL CID case.

Reply:
Correct. Need to state that correctly. The point is, in the absence of CID, implementations must track the path over which the packet loss or the RTT measurement occurred, because the number space is shared.

Reply:
Fixed, per next comment.